mergekit/scripts/mixtral_moe.py

```diff
@@ -366,6 +366,7 @@ def build(
     )
     tokenizer.padding_side = "left"
     tokenizer.pad_token_id = tokenizer.bos_token_id
+    tokenizer.pad_token = tokenizer.eos_token
```

Comment:
It'd be great to make this conditional - if the pad token is already set it'd be good to not overwrite it.